src/libsyntax_pos/hygiene.rs

```diff
@@ -550,7 +550,7 @@ impl Span {
     /// The returned span belongs to the created expansion and has the new properties,
     /// but its location is inherited from the current span.
     pub fn fresh_expansion(self, expn_data: ExpnData) -> Span {
-        self.fresh_expansion_with_transparency(expn_data, Transparency::SemiTransparent)
+        self.fresh_expansion_with_transparency(expn_data, Transparency::Transparent)
     }
 
     pub fn fresh_expansion_with_transparency(
```

Comment:
At this point we can probably merge `fresh_expansion_with_transparency` -> `fresh_expansion` and always pass the transparency explicitly.

Or perhaps remove it entirely in favor of `with_def_site_ctxt`/`with_call_site_ctxt`.

Reply:
https://github.com/rust-lang/rust/pull/63919#discussion_r318489144